src/components/Campaign.tsx

```diff
@@ -55,14 +55,24 @@ export const Campaign = () => {
   const redirectedFromSummary =
     searchParams.get("redirectedFromSummary")?.toUpperCase() === "TRUE";
 
+  const idABTest = searchParams.get("idABTest");
+
   const nextUrl = redirectedFromSummary
-    ? `${dopplerLegacyBaseUrl}/Campaigns/Summary/Index?IdCampaign=${idCampaign}`
-    : `${dopplerLegacyBaseUrl}/Campaigns/Recipients/Index?IdCampaign=${idCampaign}` +
+    ? `${dopplerLegacyBaseUrl}/Campaigns/Summary/${
+        idABTest ? "TestAB" : "Index"
```

Comment:
What do you think about calculating this before to simplify the code and reuse?

Something like:

```js
const lastSegment = idABTest ? "TestAB" : "Index";
const fixedIdCampaign = idABTest ? idABTest : idCampaign;

const nextUrl = redirectedFromSummary
    ? `${dopplerLegacyBaseUrl}/Campaigns/Summary/${lastSegment}?IdCampaign=${fixedIdCampaign}`
    : `${dopplerLegacyBaseUrl}/Campaigns/Recipients/${lastSegment}?IdCampaign=${fixedIdCampaign}` +
      `&RedirectedFromSummary=False&RedirectedFromTemplateList=False`;
```

Reply:
Sounds good. Fixed!